mix.exs

```diff
@@ -22,7 +22,8 @@ defmodule Hexpm.Mixfile do
   end
 
   defp elixirc_paths(:test), do: ["lib", "test/support"]
-  defp elixirc_paths(_),     do: ["lib"]
+  defp elixirc_paths(:dev), do: ["lib", "test/support/fake.ex", "test/support/factory.ex"]
```

Comment:
I'm not a huge fan of this. I need this to make seeds.exs work since I converted it to use ex_machina now. I don't want to put Factory and Fake in lib/ and I don't think I want to introduce a new top level directory for just these two files.

@wojtekmach Any ideas?